openedx/core/djangoapps/auth_exchange/tests/test_forms.py

```diff
@@ -38,7 +38,6 @@ def _assert_error(self, data, expected_error, expected_error_description):
             form.errors,
             {"error": expected_error, "error_description": expected_error_description}
         )
-        self.assertNotIn("partial_pipeline", self.request.session)
```

Comment:
Can we restore a line that has the same effect as this? We'd need to do something like this:

```python
from third_party_auth.pipeline import get as get_tpa_pipeline
#...
self.assertIsNone(get_tpa_pipeline(self.request))
```